spec/integration/previews_a_url_when_editing_title_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe "Shows a preview of the URL", type: :feature, js: true do
```

Comment:
These are now in /features - can you move it? Also, you can lose the 'type: :feature' bit and change 'RSpec.describe' to 'RSpec.feature'.